[2024-12-06T00:25:24.262+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T00:25:24.691+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T23:00:00+00:00 [queued]>
[2024-12-06T00:25:24.989+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T23:00:00+00:00 [queued]>
[2024-12-06T00:25:25.192+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-06T00:25:25.957+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): collect_data> on 2024-10-15 23:00:00+00:00
[2024-12-06T00:25:26.269+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T00:25:26.366+0000] {standard_task_runner.py:72} INFO - Started process 2247 to run task
[2024-12-06T00:25:26.294+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'class_act_7_pipeline', 'collect_data', 'scheduled__2024-10-15T23:00:00+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/***_pipeline.py', '--cfg-path', '/tmp/tmpnsgv1xrz']
[2024-12-06T00:25:26.413+0000] {standard_task_runner.py:105} INFO - Job 264: Subtask collect_data
[2024-12-06T00:25:28.213+0000] {task_command.py:467} INFO - Running <TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T23:00:00+00:00 [up_for_retry]> on host cff371549a6e
[2024-12-06T00:25:31.960+0000] {job.py:229} INFO - Heartbeat recovered after 10.86 seconds
[2024-12-06T00:25:32.468+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='abdulrahmanbutt7273@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='class_act_7_pipeline' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-15T23:00:00+00:00'
[2024-12-06T00:25:32.695+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-06T00:25:32.763+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-06T00:25:32.822+0000] {logging_mixin.py:190} INFO - Current task name:collect_data state:up_for_retry start_date:2024-12-06 00:25:24.737432+00:00
[2024-12-06T00:25:32.872+0000] {logging_mixin.py:190} INFO - Dag name:class_act_7_pipeline and current dag run status:running
[2024-12-06T00:25:32.888+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T00:25:34.237+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T00:25:34.359+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T00:25:34.414+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=class_act_7_pipeline, task_id=collect_data, run_id=scheduled__2024-10-15T23:00:00+00:00, execution_date=20241015T230000, start_date=20241206T002524, end_date=20241206T002534
[2024-12-06T00:25:34.804+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-06T00:25:34.822+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-06T00:25:34.841+0000] {logging_mixin.py:190} INFO - Dag name:class_act_7_pipeline queued_at:2024-12-05 00:24:08.129313+00:00
[2024-12-06T00:25:34.859+0000] {logging_mixin.py:190} INFO - Task hostname:cff371549a6e operator:PythonOperator
[2024-12-06T00:25:35.123+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T00:25:35.331+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-06T00:28:37.948+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T00:28:38.267+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T23:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-12-06T00:28:38.347+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
